src/compiler/checker.ts

```diff
@@ -13422,6 +13669,11 @@ namespace ts {
                 // in arguments too early. If possible, we'd like to only type them once we know the correct
                 // overload. However, this matters for the case where the call is correct. When the call is
                 // an error, we don't need to exclude any arguments, although it would cause no harm to do so.
+                if (isJsxOpeningOrSelfClosingElement) {
+                    // If there is not result, just return the last one we try as a candidate.

```

Comment:
1. It looks like this `if` should go before the preceding comment block that talks about `excludeArgument`.
2. I would delete the first line of the comment because it just repeats what the code itself says.